waypoints: {
    door_1_lab_lab : [[2.834, -1.917, 0], [ 0, 0, -0.437, 0.899]],
    door_1_lab_kitchen : [[2.797, -1.931, 0], [ 0, 0, 0.868, 0.497]],
    door_1_kitchen_kitchen : [[4.206, -3.576, 0], [ 0, 0, 0.912, 0.411]],
    door_1_kitchen_lab : [[4.300, -3.640, 0], [ 0, 0, -0.498, 0.867]],
    sink_1 : [[0.174, -5.906, 0], [0, 0, -0.517, 0.856]],
    charger_1 : [[-0.612, 0.868, 0], [ 0, 0, 0.257, 0.966]],
    test : [[1, 0, 0], [0, 0, 0, 1]]
}
